src/resources/triton_python_backend_utils.py

```diff
@@ -498,6 +500,36 @@ def add_output(self, output):
         self._model_config["output"].append(output)
 
 
+def log(message, level, nested_call=False):
```

Comment:
Now that there is a way to extract the line number and file information, can we move this logic to the C++ side?